app/views/case_study/show.html.erb

```diff
@@ -0,0 +1,70 @@
+<% content_for :title, "#{@case_study_presenter.page_title} - #{I18n.t("formats.#{content_item.document_type}", count: 1)} - GOV.UK" %>
+
+<% content_for :extra_headers do %>
+  <%= render "govuk_publishing_components/components/machine_readable_metadata", { content_item: content_item.to_h, schema: :article } %>
```

Comment:
@sairamya93 and I were just checking the metadata and noticed the meta `description` tag needs to be included here, similar to https://github.com/alphagov/frontend/pull/4509/files#diff-ee902e589d40510716bed23cb80d0a765570d0867944a3a9ea98cffcf34fa1afR10